db/migrate/20200528113844_add_deleted_at_indexes_to_user_related_models.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtIndexesToUserRelatedModels < ActiveRecord::Migration[5.2]
+  def change
+    add_index :users, :deleted_at
+    add_index :phone_number_authentications, :deleted_at
```

Comment:
Ah, I see we missed a few. Do we need to be safe and run this migration concurrently?

Reply:
Made them all concurrent.